quickvision/datasets/classification.py

```diff
@@ -40,26 +40,31 @@ class CSVDataset(Dataset):
     """
```

Comment:
Can this go under classification folder?

Reply:
Yes, it can since this is for single-label classification.